package-lock.json

```diff
@@ -1,8 +1,18762 @@
 {
   "name": "os2iot-frontend",
   "version": "1.0.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```

Comment:
Reset this as no dependency changes were made

Reply:
Thank you for the comment. The file has been reverted.